eng/pipelines/templates/BuildExtension.yml

```diff
@@ -0,0 +1,151 @@
+parameters:
```

Comment:
Rename the file to `build-extension.yml` to match the naming style of the other files.